types/k6/encoding.d.ts

```diff
@@ -7,8 +7,24 @@
  * @example
  * encoding.b64decode(str)
  * encoding.b64decode(str, 'rawstd')
+ * const decBuffer = encoding.b64decode(str, 'rawurl')
+ * let decBin = new Uint8Array(decBuffer);
  */
-export function b64decode(input: string | ArrayBuffer, encoding?: Base64Variant): string;
+export function b64decode(input: string, encoding?: Base64Variant): ArrayBuffer;
+
+/**
+ * Base64 decode a string.
+ * https://k6.io/docs/javascript-api/k6-encoding/b64decode-input-encoding
+ * @param input - Base64 encoded string or ArrayBuffer object.
+ * @param encoding - Base64 variant.
+ * @returns Decoded string.
+ * @example
+ * encoding.b64decode(str)
+ * encoding.b64decode(str, 'rawstd')
+ * const decBuffer = encoding.b64decode(str, 'rawurl')
+ * let decBin = new Uint8Array(decBuffer);
```

Comment:
The `format` parameter isn't described here or in the example

Reply:
should be fixed by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53101/commits/daf9b4e88aede66d57bc0721af49ec5c95616c89